sktime/utils/datetime.py

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
+"""
+pandas.datetime functions related utilities.
+
+modified the _get_freq function
+"""
 
 __author__ = ["Markus Löning"]
```

Comment:
you should add yourself to the authors (GitHub ID) and change markus to `"mloning"`

Reply:
Thanks, will add my name to the author list. (just in alignment with the author list of other files as well)